enterprise/admin/views.py

```diff
@@ -508,7 +513,12 @@ def notify_enrolled_learners(cls, enterprise_customer, request, course_id, users
                 ).format(course_id)
             )
             return
-        course_url = course_details.get('marketing_url')
+
+        marketing_site_enabled = configuration_helpers.get_value(
+            'ENABLE_MKTG_SITE',
+            settings.FEATURES.get('ENABLE_MKTG_SITE', False)
+        )
+        course_url = course_details.get('marketing_url') if marketing_site_enabled else None
```

Comment:
Yes, you were correct. The case when `course_url` is None was not covered before your change here which is why the tests were passing (we never hit the `reverse` call). The think the way forward here is actually to not send them to the `course_about` URL at all and instead construct the [course info URL](https://courses.edx.org/courses/course-v1:MITx+CTL.SC3x+2T2017/course/) without using the Django reverse function. For program enrollment, the link should take them to the [program detail dashboard in LMS](https://courses.edx.org/dashboard/programs/f5448140-88fc-451c-82cf-976504bdfa8d/).